.circleci/config.yml

```diff
@@ -1047,15 +1047,15 @@ workflows:
           requires:
             - dev-upload-docker
 
-#  nightly-acceptance-tests-consul:
-#    triggers:
-#      - schedule:
-#          cron: "0 0 * * *"
-#          filters:
-#            branches:
-#              only:
-#                - main
-#    jobs:
-#      - acceptance-kind-1-23-consul-nightly-1-11
-#      - acceptance-kind-1-23-consul-nightly-1-12
-#      - acceptance-kind-1-23-consul-nightly-1-13
+  nightly-acceptance-tests-consul:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
+    jobs:
+      - acceptance-kind-1-23-consul-nightly-1-11
+      - acceptance-kind-1-23-consul-nightly-1-12
+      - acceptance-kind-1-23-consul-nightly-1-13
```

Comment:
Is this meant to be enabled? The PR title says only 1.11 and 1.12. We're also running consul 1.13 by default in all tests

Reply:
The title has been updated.
We are not running consul-1.13 nightly images elsewhere are we?

Reply:
We are running them everywhere else (on every PR and in the nightly pipeline) as that is now the default consul version in the helm chart. We're using the latest dev consul-k8s image though which I think makes sense for 1.13.